docs/defaults.md

```diff
@@ -0,0 +1,26 @@
+## Default values
```

Comment:
I agree. This page, docs/README.md, and docs/configuration.md should probably all live in the original readme